cross/python3/patches/005-configure.ac-libffi.patch

```diff
@@ -0,0 +1,37 @@
+Ensures that the LIBFFI_INCLUDEDIR environment variable can be forced
+
+*** configure.ac.orig	2020-04-13 14:50:17.476803111 +0000
+--- configure.ac	2020-04-13 14:52:29.859184113 +0000
+***************
+*** 3011,3022 ****
+      with_system_ffi="yes"
+  fi
+  
+! if test "$with_system_ffi" = "yes" && test -n "$PKG_CONFIG"; then
```

Comment:
The default python mechanism is to use pkg configuration to locale libffi binaries.
The only issue here is that libffi binaries are installed with information which actually corresponds to the target installation not the build installation.
So I basically needed to override it using environment variable.